docs/products/flink/concepts/flink-applications.rst

```diff
@@ -0,0 +1,56 @@
+Aiven for Apache Flink® applications
+====================================
+
+An Aiven for Apache Flink® Application is an abstraction layer on top of Apache Flink SQL that includes all the elements related to a Flink job to help building your data processing pipeline. An application contains the definition of source and sink tables, data processing logic, deployment parameters, and other necessary metadata. 
+
+Applications are the starting point for running an Apache Flink job within the Aiven managed service. The `Aiven Console <https://console.aiven.io/>`_ provides a user-friendly, guided wizard to help you build and deploy applications, create source and sink tables, write transformation statements, and validate and ingest data using the interactive query feature.
+
+Each application created and deployed in the Aiven for Apache Flink service performs a specific data transformation task allowing you to map and manage different workflows separately within the Aiven for Apache Flink service. For example, you can create an application to get user IDs from one topic and publish them on another topic. You can run multiple applications within the Aiven for Apache Flink service.
+
+Applications significantly improve the developer experience and simplify the development and deployment of Flink applications. Applications are **automatically versioned** on every edit of the underline definition (tables, or transformation SQL) allowing you to eperiment new transformation and revert to a previously stored definition if the result of the edits doesn't meet expectations. 
+
+Features of Aiven for Apache Flink applications
+-----------------------------------------------
+
+Some of key feature of Aiven for Flink applications include: 
+
+* **Intuitive Interface**: The `Aiven Console <https://console.aiven.io/>`_ provides a user-friendly, guided wizard to help you quickly build and deploy your applications.
+* **Source and sink tables** definition with **SQL autocomplete**: You can create source and sink tables, the SQL autocomplete feature allows you to start from a specific set of pre-filled parameters depending on the type of connector.
+* **Interactive query**: The interactive query feature allows you to validate and preview your data in the table or SQL tranformation before deploying it.
+* **Start and stop**: you can start and stop applications stop and start anytime.
+* **Automatic Versioning**: on every edit of the table or data transformation definition, a new application versions is automatically generated allowing you to keep track of the changes and revert to a previous version if necessary.
+* **Savepoints**: You can stop your application with a savepoint, allowing you to restart it from the previous state at a later time.
+* **Scalability:** You can scale your cluster depending on the demands of your specific application by parallelizing the applications into multiple tasks and executing them concurrently in a cluster.
```

Comment:
I believe we need to discuss this in more detail:
* Scalability of the cluster is about Apache Flink itself (not at application level)
* Change the level of parallelism is at application level, but needs a separate discussion

Reply:
Got it, thanks. Updated the text.